frontend/src/app/components/crm/analytics/analytics.component.spec.ts

```diff
@@ -20,8 +20,8 @@ describe('AnalyticsComponent', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
-
+/*
```

Comment:
Instead of commenting this use [xit](https://jasmine.github.io/api/2.7/global.html#xit) to ignore a test.